using-mobile-extensions/mobile-core/README.md

```diff
@@ -1,22 +1,18 @@
 # Mobile Core
 
-The **Mobile Core** represents the core Adobe Experience Platform SDK that is required for every app implementation. The core contains a common set of functionality and frameworks such as an Experience Cloud Identity services, data event hub, rules engine, reusable networking, disk access routines, and so on, which is required by all Adobe and third-party extensions. The following sections provide additional information about what is provided in the Mobile Core extension. 
+The **Mobile Core** represents the core Adobe Experience Platform SDK that is required for every app implementation. The core contains a common set of functionality and frameworks such as an Experience Cloud Identity services, data event hub, rules engine, reusable networking, disk access routines, and so on, which is required by all Adobe and third-party extensions. The following sections provide additional information about what is provided in the Mobile Core extension.
 
-## Configuration methods
-
-For more information, see [Configuration Methods Reference](configuration-reference/).
-
-## Event hub APIs
+## Mobile Core generic APIs
 
 The SDK provides core, extension-independent APIs to facilitate the event tracking of user screens, actions, and PII data. Events generated by these APIs are published to the SDK event hub and are available for use by extensions. For instance, when the Analytics extension is installed, all user actions and app screens event data will be sent to appropriate Analytics reporting endpoints.
 
-For more information, see [Mobile Core API Reference](configuration-reference/mobile-core-api-reference.md).
+For more information, see [Mobile Core API Reference](mobile-core-api-reference.md) and [Mobile Core Event Reference](mobile-core-event-reference.md).
 
-## Rules engine
+## Configuration
 
-This Mobile Core contains a rule engine that interprets all tracking, PII, Signals, _Request Content_, and extension-specific events for the SDK. Users can create rules, which when matched, trigger a _Response Content_ event for the listening extension to handle the associated consequence\(s\).
+The Configuration extension will generate the necessary events in the Event Hub that will provide configuration details for all installed extensions.
```

Comment:
Please change "extension will generate" to "generates" and change "that will provide" to "provide", and add "the" between "provide" and "configuration". The sentence should read as follows:

"The configuration extension provides the necessary events in the Event Hub that provide the configuration details for all installed extensions."